### ISSUE TITLE
Inference result does not match with the reported result

### ISSUE THREAD

**Issue by oaishi** [facebookresearch/synsin#26]
Dear authors,

When I tried to infer the model locally on RealEstate10k with the provided pretrained model, the accuracy difference was very large from the ones mentioned in the paper. As mentioned in the [Readme](https://github.com/facebookresearch/synsin#usage), I understand the reason of this issue. But as I am working on a variant of Synsin and a smaller subset of RealEstate10k, I needed to report the result obtained using Pytorch3D distribution. Can you kindly suggest me how I should proceed?

I ran the inference on Windows OS on GTX 1070 GPU with batch size 1. My python version is 3.8 and pytorch version is 1.5.1. I saved the extracted images as 256*256  in the disk.

**Comment by oawiles** [comment]
I would first check you get the same result as the pre saved result in the demo notebooks. If you do get the same result then the model is loading in right, else it could be a version issue.

**Comment by oawiles** [comment]
Also first visually inspect the results to ensrue there are no obvious issues.

**Comment by oaishi** [comment]
Thank you so much for your reply! I visually inspected the results, they mostly looked pretty well. I would compare the result in the pre-loaded demos and let you know if I face any other issue. Thanks again for taking the time and replying!
